modules/monitoring-configuring-audit-logs-for-metrics-server.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: CONCEPT
```

Comment:
This module is a concept, but it seems to be written as a procedure in which the step(s) is to edit a config map. You might consider changing this to a procedure module.

Reply:
Hello @bhardesty thank you for this observation. I am intentionally including this as a conceptual module in order to avoid using the necessary procedure schema.

I chose to include this as a concept because there is this exception mentioned in the [modular docs](https://redhat-documentation.github.io/modular-docs/#creating-concept-modules):

> "Avoid including instructions to perform an action. Action items belong in procedure modules. However, in some cases a concept or reference module can include suggested actions **when those actions are simple**,  are highly dependent on the context of the module, and have no place in any procedure module."

Thank you!